src/app/teacher/manage-tags-dialog/manage-tags-dialog.component.html

```diff
@@ -0,0 +1,60 @@
+<h2 mat-dialog-title i18n>Manage Tags</h2>
+<mat-dialog-content class="dialog-content-scroll">
+  <div fxLayout="column" fxLayoutGap="12px">
+    @if (!showCreateTag) {
+      <div>
+        <button mat-flat-button color="primary" (click)="showCreateTag = !showCreateTag" i18n>
+          + New Tag
+        </button>
+      </div>
+    }
+    @if (showCreateTag) {
+      <edit-tag (closeEvent)="showCreateTag = false" />
+    }
```

Comment:
Switch test order and use ```@else``` for better readability?
```suggestion
    @if (showCreateTag) {
      <edit-tag (closeEvent)="showCreateTag = false" />
    } @else {
      <div>
        <button mat-flat-button color="primary" (click)="showCreateTag = !showCreateTag" i18n>
          + New Tag
        </button>
      </div>
    }
```